src/menus/menu.jsx

```diff
@@ -547,7 +544,7 @@ const Menu = React.createClass({
           transitionDelay = cumulativeDelay;
         }
 
-        childrenContainerStyles = this.mergeStyles(styles.menuItemContainer, {
+        childrenContainerStyles = Object.assign(styles.menuItemContainer, {
```

Comment:
Thanks missed the loop that time, though it didn't seem to cause an issue during testing.

I'm going to add it anyway though.